data/AppConfig/base-configuration.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
make a configs folder and place this file in that folder. Please change the name of AppConfig to app_config
path would  become data/configs/app_config/base-configuration.json.